E2E/FirstTest.spec.js

```diff
@@ -7,5 +7,6 @@ test('My first Test', async ({ page }) => {
     await page.locator('#APjFqb').click();
     await page.locator('#APjFqb').fill('Globo');
```

Comment:
Put the locator in an variable and use this variable on lines 7 and 8

Reply:
Done!